package.json

```diff
@@ -17,6 +17,7 @@
         "is-invalid-path": "^1.0.2",
         "mocha": "5.2.0",
         "prompts": "^2.2.1",
+        "simple-git": "^1.132.0",
```

Comment:
Please remove all `^` so we will use the version without an automatic upgrade. I have missed removing other `^`